ci-jobs/crowdin-common.js

```diff
@@ -0,0 +1,50 @@
+const { logger } = require('@appium/support');
+const axios = require('axios');
+const path = require('path');
+const _ = require('lodash');
```

Comment:
Can we use ES6 `import`/`export` here and in the other 2 files?

Reply:
Not sure how to enforce that in raw node.js. I usually rely on babel or TS transpilers

Reply:
nvm, found a tutorial on manual CJS->ESM conversion